### PR TITLE
Updated “Powered by” footer images

### PR DESCRIPTION
**Pull request by DustinHartzler**
## Description

Replaces “WordPress.com” with “WordPress” in the footer images. Added a cache busting string so all images will be updated more quickly

### Updated Footer Images

We updated our footer branding images. All images remain the same size / aspect ratio.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

The documentation will be updated with new images once the PR has been merged.

### CODE REVIEW

**Comment by rebeccahum** [comment]
@DustinHartzler Can you fix the linting errors please?

File: vip-helpers/vip-utils.php [hunk 1]
```diff
@@ -84,7 +87,7 @@ function vip_powered_wpcom_img_html( $image ) {
 	);
 
 	if ( array_key_exists( $image, $vip_powered_wpcom_images ) ) {
-		return '<a href="' . esc_url( vip_powered_wpcom_url() ) . '" rel="generator nofollow" class="powered-by-wpcom"><img src="' . esc_url( plugins_url( 'images/' . $vip_powered_wpcom_images[ $image ][0], __FILE__ ) ) . '" width="' . esc_attr( $vip_powered_wpcom_images[ $image ][1] ) . '" height="' . esc_attr( $vip_powered_wpcom_images[ $image ][2] ) . '" alt="' . esc_attr__( 'Powered by WordPress VIP', 'vip-helpers' ) . '" /></a>';
+		return '<a href="' . esc_url( vip_powered_wpcom_url() ) . '" rel="generator nofollow" class="powered-by-wpcom"><img src="' . esc_url( plugins_url( 'images/' . $vip_powered_wpcom_images[ $image ][0], __FILE__ ) ) . '?ver=' . $IMG_VERSION . '" width="' . esc_attr( $vip_powered_wpcom_images[ $image ][1] ) . '" height="' . esc_attr( $vip_powered_wpcom_images[ $image ][2] ) . '" alt="' . esc_attr__( 'Powered by WordPress VIP', 'vip-helpers' ) . '" /></a>';
```
Comment (rebeccahum) [inline vip-helpers/vip-utils.php:90]: Please late escape `$IMG_VERSION` with `rawurlencode()`.

File: vip-helpers/vip-utils.php [hunk 2]
```diff
@@ -73,6 +73,9 @@ function wpcom_vip_noncdn_uri( $path ) {
  * @return string HTML
  */
 function vip_powered_wpcom_img_html( $image ) {
+	//Stores constant to allow cache breaking if images need replaced
+	$IMG_VERSION = "20220317";
```
Comment (jesse-greathouse) [inline vip-helpers/vip-utils.php:77]: should be:

```$img_version = '20220317'```
according to the coding standards of the project.

Change this and the other use of the variable and it should pass linting.

File: vip-helpers/vip-utils.php (hunk 1 above)
Comment (DustinHartzler) [reply vip-helpers/vip-utils.php]: Thanks for the reminder!

File: vip-helpers/vip-utils.php (hunk 2 above)
Comment (DustinHartzler) [reply vip-helpers/vip-utils.php]: Thanks Jesse. Updated both variables and changed the comment.

File: vip-helpers/vip-utils.php [hunk 3]
```diff
@@ -84,7 +87,7 @@ function vip_powered_wpcom_img_html( $image ) {
 	);
 
 	if ( array_key_exists( $image, $vip_powered_wpcom_images ) ) {
-		return '<a href="' . esc_url( vip_powered_wpcom_url() ) . '" rel="generator nofollow" class="powered-by-wpcom"><img src="' . esc_url( plugins_url( 'images/' . $vip_powered_wpcom_images[ $image ][0], __FILE__ ) ) . '" width="' . esc_attr( $vip_powered_wpcom_images[ $image ][1] ) . '" height="' . esc_attr( $vip_powered_wpcom_images[ $image ][2] ) . '" alt="' . esc_attr__( 'Powered by WordPress VIP', 'vip-helpers' ) . '" /></a>';
+		return '<a href="' . esc_url( vip_powered_wpcom_url() ) . '" rel="generator nofollow" class="powered-by-wpcom"><img src="' . esc_url( plugins_url( 'images/' . $vip_powered_wpcom_images[ $image ][0], __FILE__ ) ) . '?ver=' . rawurlencode($img_version) . '" width="' . esc_attr( $vip_powered_wpcom_images[ $image ][1] ) . '" height="' . esc_attr( $vip_powered_wpcom_images[ $image ][2] ) . '" alt="' . esc_attr__( 'Powered by WordPress VIP', 'vip-helpers' ) . '" /></a>';
```
Comment (rebeccahum) [inline vip-helpers/vip-utils.php:90]: We encourage using the space usage in accordance with the WordPress Coding Standards:  https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage

**Review by rebeccahum — CHANGES_REQUESTED** [review]

File: vip-helpers/vip-utils.php (hunk 3 above)
Comment (DustinHartzler) [reply vip-helpers/vip-utils.php]: I knew that...updated now. Thanks for the feedback.

File: vip-helpers/vip-utils.php [hunk 4]
```diff
@@ -73,6 +73,9 @@ function wpcom_vip_noncdn_uri( $path ) {
  * @return string HTML
  */
 function vip_powered_wpcom_img_html( $image ) {
+	//Stores string to allow cache breaking if images need replaced
+	$img_version = "20220317";
```
Comment (rebeccahum) [inline vip-helpers/vip-utils.php:77]: Minor nitpick, can we move this closer to line 89 since that's where it'll be used?

Comment (DustinHartzler) [reply vip-helpers/vip-utils.php]: 👍  - that's a really good suggestion. Updated.

**Comment by rebeccahum** [comment]
@DustinHartzler One remaining linting error and then we should be good to go 🙂

**Comment by DustinHartzler** [comment]
Clearly I need to check out my linting setup in VSCode. Thanks @rebeccahum

**Pull request merged**